extension/ts/extensions/pdf.ts

```diff
@@ -17,16 +15,17 @@ const onClick = (comments: HTMLTextAreaElement) => async (event: MouseEvent) =>
 	const author = findAuthor();
 	const title = findTitle();
 
-	const {overlay, style} = createLoader();
```

Comment:
I didn't like how the client was responsible for handling this `style` thing, which is really part of the loader, hence the axe in the latest commit

Reply:
🪓  makes sense